api/models/containerimageinfo.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
+
+// GetRepoTag returns the first repo tag if it exists. Otherwise, returns false.
+func (c *ContainerImageInfo) GetRepoTag() (string, bool) {
+	var tag string
+	var ok bool
+
+	if len(*c.RepoTags) > 0 {
+		tag, ok = (*c.RepoTags)[0], true
+	}
+
+	return tag, ok
+}
+
+// GetRepoDigest returns the first repo digest if it exists. Otherwise, returns false.
+func (c *ContainerImageInfo) GetRepoDigest() (string, bool) {
+	var digest string
+	var ok bool
+
+	if len(*c.RepoDigests) > 0 {
```

Comment:
need to check for c == nil && c.RepoDigests == nil ?

Reply:
@fishkerez Same as above.